src/shared/components/annotation/styles/civicCard.scss

```diff
@@ -0,0 +1,172 @@
+.civic-card {
```

Comment:
This almost looks like `oncoKbCard.scss` (which is mostly legacy code that needs to be cleaned up). I don't think we really need to duplicate everything for Civic. For example there are styles for `qtip` in this file that we don't really use anymore. It would be nicer to copy only the related styles, not the entire `oncoKbCard.scss`.